artifacts/deployment/03-sample-apiserver-deployment.yaml

```diff
@@ -31,7 +38,7 @@ spec:
       - name: sample-apiserver
         image: xmudrii/kube-sample-apiserver
         imagePullPolicy: Always
-        command: ["/kube-sample-apiserver", "--etcd-servers=http://etcd-k8s-sample-apiserver.kube-apiserver-storage.svc:2379"]
+        command: ["/kube-sample-apiserver", "--etcd-servers=https://etcd-sample-apiserver.kube-apiserver-storage.svc:2379", "--etcd-cafile=/etc/etcd-certs/ca/ca.pem", "--etcd-certfile=/etc/etcd-certs/client/client.pem", "--etcd-keyfile=/etc/etcd-certs/client/client-key.pem"]
```

Comment:
make followup to use the kube tls secret type and names.

Reply:
Created issue #43.